cypress/utils/plugins/index-management-dashboards-plugin/commands.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BACKEND_BASE, IM_API, IM_CONFIG_INDEX } from '../../constants';
+
+Cypress.Commands.add('deleteAllIndices', () => {
```

Comment:
Seems this could be a general purpose command based on the name - could some of the logic be refactored into a common command and referenced in here, and maybe rename this command to something ISM-specific? At least the renaming could prevent conflicts in the future.

Reply:
Refactor all the general command that could be used by other plugin to the main command file